package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "deploy": "gh-pages -d dist",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "server": "node server.js",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Aseadd/ToDo-List.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Aseadd/ToDo-List/issues"
+  },
+  "homepage": "https://github.com/Aseadd/ToDo-List",
```

Comment:
this is minor but, the package.json is not properly inited. what you can do is delete everything except de `devDependencies` and `dependencies` and run again `npm init -y`. ensure your node_module folder is not present.

Reply:
Done with new init